modules/end-user-portal/src/Pages/Search/Results.cshtml.cs

```diff
@@ -58,4 +72,12 @@ public async Task<IActionResult> OnGet()
         Source = DetailViewSource.Search,
         Mode = Mode
     };
+    
+    public int NumPages => (int)Math.Ceiling((decimal)SearchResult.TotalResults / _config.Value.ResultsPerPage);
```

Comment:
Can we put these properties in ResultsModel.cs

Reply:
Sure

Reply:
Moved the values not used in the .cshtml file